contents/handbook/growth/marketing/index.md

```diff
@@ -71,18 +73,20 @@ But we think that the opportunity to build a strong brand off the back of these
 
 ## What do we actually do then?
 
-The Marketing team's main focus is acquisition. Our specific objective is to _get as many relevant developers and technical product managers as possible to visit posthog.com_. 
+The Marketing team's main focus is acquisition of [high quality signups](https://posthog.com/handbook/strategy/strategy#what-should-we-be-working-on-today). We have established that _word of mouth_ is the most effective way to do this - virtually all of our high quality signups have come in organically. 
 
-We have identified the following acquisition channels as most important to our strategy. You can find out more details on their relevant pages:
+We have identified the following acquisition channels as most important to our strategy. They are listed in descending order of priority:
```

Comment:
```suggestion
We have identified the following acquisition channels as most important to our strategy. ```

Since you call out paid ads anyway, I am neutral if content / useful products are better. In fact, I could see an argument the other way around - content helps you use the products, hence products come first. However, not sure we need to distinguish them yet.

In future if product launches go great, then we could re order (or the other way around as we get more data on how content performs over time)